charts/postgresql/templates/astronomer-bootstrap-secret.yaml

```diff
@@ -11,5 +11,9 @@ metadata:
     argocd.argoproj.io/sync-wave: "-1"
   {{- end }}
 data:
+  {{- if .Values.global.pgbouncer.enabled }}
+  connection: {{ printf "postgres://postgres:postgres@astronomer-pgbouncer.astronomer.svc.cluster.local:5432" | b64enc | quote }}
```

Comment:
we need to remove hardcoding here - there are customers who use different release name

Reply:
it's only for testing, but i'll change it with values.

Reply:
no probs then, i thought we missed